desktop/core/src/desktop/js/utils/hooks/useLoadData.ts

```diff
@@ -22,6 +22,8 @@ export type IOptions<T, U> = {
   params?: U;
   fetchOptions?: ApiFetchOptions<T>;
   skip?: boolean;
+  onSuccess?: (data: T) => void;
+  onError?: (error: Error) => void;
 };
 
 type IUseLoadData<T> = {
```

Comment:
Nit: I think I mention this is some earlier preview. In the Hue codebase we don't prepend capital I to indicate an interface. Instead we simply declare as interface instead of type. Having an inconsistent usage of capital "I" to indicate interfaces will only add confusion.